WalletWasabi.Daemon/Rpc/WasabiJsonRpcService.cs

```diff
@@ -188,6 +188,24 @@ public object GetStatus()
 		};
 	}
 
+	[JsonRpcMethod("getcoinjoinstatus")]
+	public string GetCoinjoinStatus()
+	{
+		var activeWallet = Guard.NotNull(nameof(ActiveWallet), ActiveWallet);
+
+		AssertWalletIsLoaded();
+		var coinJoinManager = Global.HostedServices.Get<CoinJoinManager>();
+		_ = coinJoinManager.TryGetWalletStatus(activeWallet.WalletName, out var walletCoinjoinStatus);
+		return walletCoinjoinStatus switch
+		{
+			CoinJoinClientState.Idle => "Idle",
+			CoinJoinClientState.InProgress => "In progress",
+			CoinJoinClientState.InSchedule => "In schedule",
+			CoinJoinClientState.InCriticalPhase => "In critical phase",
+			_ => "Not managed"
```

Comment:
This makes no sense. There is not such state and nobody will never know what "Not managed" means.